mining.md

```diff
@@ -367,11 +367,23 @@ Decay = e^(ln(0.5) / (HalvingPeriodBlocks / AdjustmentPeriod))
 
 ```
 // Given one block every 30 seconds, this is how many blocks are in six years
-HalvingPeriodBlocks = 6 * 365 * 24 * 60 * 2
+HalvingPeriodBlocks = 6 * 365 * 24 * 60 * 2 = 6,307,200 blocks
 ```
 
 Note: Due to jitter in EC, and the gregorian calendar, there may be some error in the issuance schedule over time. This is expected to be small enough that it's not worth correcting for. Additionally, since the payout mechanism is transferring from the network account to the miner, there is no risk of minting *too much* FIL.
 
+### Notes on Block Reward Application
+
+As mentioned above, every round, a miner checks to see if they have been selected as the leader for that particular round (see [Secret Leader Election](expected-consensus.md#secret-leader-election) in the Expected Consensus spec for more detail). Thus, it is possible that multiple miners may be selected as winners in a given round, and thus, that there will be multiple blocks produced at the same block height (forming a TipSet). Each of the winning miners will apply the block reward directly to their actor's state in their state tree. 
+
+Once a TipSet is produced and propagated, other nodes on the network will validate the TipSet. The full procedure for how to verify a TipSet can be found above in [Block Validation](#block-validation). At a high level, in order to validate a set of blocks in a TipSet, the validating node will first apply the block reward value directly to each of the parent nodes' accounts and then apply the messages to the resulting state. Thus, each of the miners who produced a block in the TipSet will receive a block reward. There will be no lockup. These rewards can be spent immediately (in the next block and later). 
```

Comment:
tipsets arent really created and propagated, individual blocks are created, and nodes that receive them form tipsets out of other blocks at the same height with the same parents.

the general algorithm for computing the state of a tipset is:

for each block in the tipset:
  apply the block reward for the miner of the block
  apply the state transition for each message in that block

The way this paragraph is worded, it sounds like you would apply all the block rewards up front, and then apply all the messages.

Reply:
Cool, will fix!